### PR TITLE
Add show hidden files button 

### LINKED ISSUE #24: How to handle hidden files?
We probably need a 'Show hidden files' entry option like gnome's file navigator does.

Screenshot from 2023-05-17 11-40-26

### PR DESCRIPTION
**Pull request by MahmoudHamdy02**
Closes #24.

* [x] Fix check icon padding
* [x] Add tests

### CODE REVIEW

**Review by KKoukiou — COMMENTED** [review]
When show hidden files is clicked - the text updates from 'Y items (X hidden)'  to `Y items (0 hidden)`


I think that's not correct behavior are hidden files property is not changing  if we show them. We should correctly count the number still. 


Also the new action (Show hidden files) should only apply to the menu of directories, not files.

**Review by KKoukiou — COMMENTED** [review]
* when right cliking on a directory and selecting 'create new directory'' which should create a new directory inside the directory we right clicked on the page crashes with:
*

**Comment by MahmoudHamdy02** [comment]
Wrong variable! Should be fixed now

File: test/check-application [hunk 1]
```diff
@@ -391,5 +397,32 @@ class TestNavigator(testlib.MachineCase):
         b.click("div.pf-c-modal-box__close")
         m.execute("sudo chattr -i /home/admin/newdir2")
 
+    def testHiddenItems(self):
+        b = self.browser
+        m = self.machine
+
+        self.restore_dir("/home/admin")
```
Comment (KKoukiou) [inline test/check-application:404]: This is redundant https://github.com/cockpit-project/cockpit/blob/main/test/common/testlib.py#L1499

**Review by KKoukiou — COMMENTED** [review]
Can you please squash all the commits apart from 'creating directory when folder is selected creates dir inside that folder'

**Review by KKoukiou — APPROVED** [review]

**Pull request merged**